litellm/integrations/traceloop.py

```diff
@@ -1,10 +1,26 @@
+import traceback
+
+
 class TraceloopLogger:
     def __init__(self):
-        from traceloop.sdk.tracing.tracing import TracerWrapper
-        from traceloop.sdk import Traceloop
+        try:
+            from traceloop.sdk.tracing.tracing import TracerWrapper
+            from traceloop.sdk import Traceloop
+
+            Traceloop.init(app_name="Litellm-Server", disable_batch=True)
+            self.tracer_wrapper = TracerWrapper()
+        except Exception as e:
+            raise Exception(
+                f"\033[Traceloop not installed, try running 'pip install traceloop-sdk' to fix this error: {e}\n{traceback.format_exc()}\033[0m"
+            )
+
+    def log_error(kwargs, response_obj, start_time, end_time):
```

Comment:
`log_error` method is not in use. The addition of this is unclear.

Reply:
@paneru-rajan thanks, so how do I log errors from litellm?

Reply:
add it here - https://github.com/BerriAI/litellm/blob/5b1f9283412661c9abf028982523f14c172f8195/litellm/utils.py#L2437